float64 px
float64 py
float64 pz
float64 qx
float64 qy
float64 qz
float64 qw
float64 trajduration
---
